packages/core/test/unit/utils/collection-ttl.test.ts

```diff
@@ -59,9 +71,9 @@ describe('CollectionTtl', () => {
 
   describe('#ttlMilliseconds', () => {
     test('converts to milliseconds', () => {
-      const cttl = new CollectionTtl(1.23);
+      const cttl = new CollectionTtl(2);
 
-      expect(cttl.ttlMilliseconds()).toEqual(1230);
+      expect(cttl.ttlMilliseconds()).toEqual(2000);
     });
```

Comment:
not sure if this was the right move, should we still allow collectionTtl to be specified as a float?